### ISSUE TITLE
Ergonomics of Information to User to Run Program

### ISSUE THREAD

**Issue by rishabco** [tylersco/Scott_Vienneau_CSCI2270_FinalProject#1]
Hey guys, great job on the program. Just a quick comment on the program from a user's perspective. 

-I had some trouble actually starting to run the program. The menu is laid out well; however, when I wanted to run the "Print Vertices Option", it prompted me to enter a starting landmark (also for some options I had to enter an ending landmark). Problem is, I had absolutely no idea what landmarks were there; I assumed I could just select print all vertices to know what I was working with. In the end I had to check the .txt file. Perhaps you could print out a list of landmarks at the beginning of the program, or hard-code a starting city for printing the vertices. 

-In terms of information is the numbers in front of the cities when they are being printed out; I am assuming they are the district ID, but anyone not familiar with the previous graph assignment may be confused with "1." being in front of all the city names. On that note, since it seems like all of the landmarks are in the same district, I don't necessarily see the point in having them.  If you want to keep them, I suggest a) making a .txt file that incorporates multiple districts, and b) include some feedback when you set the districts (right now setting the districts prints nothing). 

-Instead "Print Vertices" and "Delete Edge", perhaps changing it to something like "Display all Landmarks" and "Delete Route" would be more appropriate in the context of your project.
- A simple cout -ed message to the user at the beginning of the program would help a user understand what is happening. 

In conclusion, I just feel that your program could use some more user printed information to tell the user what is going on. Luckily, all of these things can be solved just by modifying/adding some cout statements to your code. It's obvious looking at your code that it is very complex and that you put a lot of work into it; I am simply suggesting that you use some statements to express that to the user.

**Comment by tylersco** [comment]
Thank you for the feedback!

I just want to clarify one thing you said: all of the landmarks start in the same district, but there is a menu option to delete edges and delete vertices and thus, when deleting edges or vertices, more districts will arise. Initially yes, all landmarks are connected, but that may not be the end case, and for that reason we added districts. 

Other than that, thank you for all the feedback and I will make some changes!

**Comment by tylersco** [comment]
I have added some user-friendly messages as well as a displayEdges method which will display all vertices and adjacent vertices so you can view all landmarks in the Graph. There is also no parameter for this method.
